app/views/layouts/application.html.slim

```diff
@@ -0,0 +1,12 @@
+doctype html
+html
+  head
+    title
+      | InstaSpace
+    = csrf_meta_tags
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
+    = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+  body
+    = flash[:notice]
+    = flash[:alert]
```

Comment:
Change it to condition statement. You can use bootstrap styling:
```
<% flash.each do |key, value| %>
     <div class="alert alert-<%= key %>"><%= value %></div>
 <% end %>
```
(in slim of course! :) )